_emails/redwoodcity.md

```diff
@@ -0,0 +1,42 @@
+subject: Redwood City Resident for Defunding the RCPD
+body: |-
+	To Mayor Howard, Vice Mayor Masur, and Redwood City City Council Members,
+
+	My name is [YOUR NAME], and I am a resident of Redwood City. This past week, our nation and community have been gripped by protests calling for an end to racism and anti-blackness and a complete overhaul in our approach to criminal justice in America. I am writing to demand real change to the Redwood City criminal justice system.
+
+	In the city’s recommended FY 2020-21 operating budget, $48.9M (34.3%) is allocated to the police, up from $46.5M (33.9%) in the FY 2019-20 budget. It is shameful and unproductive that, in comparison, the recommended budget only allocates $6.8M to affordable housing. Research shows that a living wage, access to holistic health services and treatment including mental health services, educational opportunity, and stable housing are far more successful at reducing crime than punitive systems like police or prisons (Source: Popular Democracy). 
+
+	The average police recruit spends 58 hours learning how to shoot and only 8 hours learning how to de-escalate (Source: Campaign Zero). They are not trained or equipped to react to the vast majority of crises. In our own city just last year, RCPD shot Kyle Hart, who was suffering a mental health crisis. Despite their de-escalation training, the two officers involved didn’t even try to de-escalate the situation, resulting in the death of a 33-year-old father of two young children. 
+
+	In the journal of Sociology of Race and Ethnicity, Phillip McHarris (PhD candidate at Yale focusing on race) argues that we must work towards a reality in which healthcare workers and emergency response teams handle substance abuse, domestic violence, homelessness, or mental health cases. Policies to “improve the police” are not enough, as there’s no evidence that implicit bias training or community relations initiatives help with reducing the abuses of policing (Sources: The Nation, The Atlantic). We need to reimagine public safety to prioritize alternatives to conflict rather than defaulting to violence.
+
+	I demand immediate reduction of the police force budget, cancellation of cadet classes, demilitarization of our forces, and reallocation of funds from police to community-led health and safety strategies. We should redirect police funding to efforts that are actually proven to reduce crime, such as affordable housing, shelters, and mental health services. Redwood City should support community wellbeing, rather than empowering the police forces that tear our community apart. Police reforms, such as de-escalation training, was not enough to save Kyle Hart and is not enough to protect our community.
+
+	It is your duty to represent your constituents. I am urging you to revise the Redwood City recommended operating budget for FY 2020-21, and to increase funds to non-punitive community efforts. 
+
+	Thank you for your time,
+
+	\[YOUR NAME\]
```

Comment:
Please remove backslashes

Reply:
Also yaml issues

Reply:
I removed backslashes, but not sure how to fix the yaml issues.

Reply:
I think they need to be  spaces and not tabs. Try it out in a yaml web editor

Reply:
Ok I *think* I addressed the issues. Let me know if you still see issues. thank you!